diff -ur webcleaner-2.29/config/adzap2wc.py webcleaner-2.29.new/config/adzap2wc.py
--- webcleaner-2.29/config/adzap2wc.py	2005-04-11 19:32:17.000000000 +0000
+++ webcleaner-2.29.new/config/adzap2wc.py	2005-04-12 09:08:28.535760600 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 # this script has to be executed in the config parent dir
 """Generate WebCleaner .zap files from an AdZapper squid_redirect file"""
diff -ur webcleaner-2.29/config/bl2wc.py webcleaner-2.29.new/config/bl2wc.py
--- webcleaner-2.29/config/bl2wc.py	2005-04-11 19:32:17.000000000 +0000
+++ webcleaner-2.29.new/config/bl2wc.py	2005-04-12 09:08:28.537760296 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 # this script has to be executed in the config parent dir
 """Generate blacklist_XYZ folders with blocking and rewriting
diff -ur webcleaner-2.29/config/dmozfilter.py webcleaner-2.29.new/config/dmozfilter.py
--- webcleaner-2.29/config/dmozfilter.py	2005-04-11 19:32:17.000000000 +0000
+++ webcleaner-2.29.new/config/dmozfilter.py	2005-04-12 09:08:28.538760144 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # filter the big dmoz file
 # Usage:
 # zcat content.rdf.u8.gz | dmozfilter.py | gzip --best > dmoz.rdf.stripped.gz
diff -ur webcleaner-2.29/config/reindent.py webcleaner-2.29.new/config/reindent.py
--- webcleaner-2.29/config/reindent.py	2005-04-11 19:32:18.000000000 +0000
+++ webcleaner-2.29.new/config/reindent.py	2005-04-12 09:08:28.539759992 +0000
@@ -1,4 +1,4 @@
-#! /usr/bin/python2.3
+#! /usr/bin/python
 
 # Released to the public domain, by Tim Peters, 03 October 2000.
 
diff -ur webcleaner-2.29/test/block.py webcleaner-2.29.new/test/block.py
--- webcleaner-2.29/test/block.py	2005-04-11 19:32:21.000000000 +0000
+++ webcleaner-2.29.new/test/block.py	2005-04-12 09:08:40.708910000 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 
 def _main ():
diff -ur webcleaner-2.29/test/exportratings.py webcleaner-2.29.new/test/exportratings.py
--- webcleaner-2.29/test/exportratings.py	2005-04-11 19:32:21.000000000 +0000
+++ webcleaner-2.29.new/test/exportratings.py	2005-04-12 09:08:40.709909848 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 
 def _main ():
diff -ur webcleaner-2.29/test/filterfile.py webcleaner-2.29.new/test/filterfile.py
--- webcleaner-2.29/test/filterfile.py	2005-04-11 19:32:21.000000000 +0000
+++ webcleaner-2.29.new/test/filterfile.py	2005-04-12 09:08:40.709909848 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 """
 Filter given .html file with Replacer, Rewriter and BinaryCharFilter module.
diff -ur webcleaner-2.29/test/get.py webcleaner-2.29.new/test/get.py
--- webcleaner-2.29/test/get.py	2005-04-11 19:32:21.000000000 +0000
+++ webcleaner-2.29.new/test/get.py	2005-04-12 09:08:40.711909544 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 """print headers of an url"""
 
diff -ur webcleaner-2.29/test/getssl.py webcleaner-2.29.new/test/getssl.py
--- webcleaner-2.29/test/getssl.py	2005-04-11 19:32:21.000000000 +0000
+++ webcleaner-2.29.new/test/getssl.py	2005-04-12 09:08:40.712909392 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 """print headers of an url"""
 
diff -ur webcleaner-2.29/test/head.py webcleaner-2.29.new/test/head.py
--- webcleaner-2.29/test/head.py	2005-04-11 19:32:22.000000000 +0000
+++ webcleaner-2.29.new/test/head.py	2005-04-12 09:08:40.712909392 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 """print headers of an url"""
 
diff -ur webcleaner-2.29/test/imagereduce.py webcleaner-2.29.new/test/imagereduce.py
--- webcleaner-2.29/test/imagereduce.py	2005-04-11 19:32:22.000000000 +0000
+++ webcleaner-2.29.new/test/imagereduce.py	2005-04-12 09:08:40.713909240 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 
 import sys
diff -ur webcleaner-2.29/test/magic.py webcleaner-2.29.new/test/magic.py
--- webcleaner-2.29/test/magic.py	2005-04-11 19:32:22.000000000 +0000
+++ webcleaner-2.29.new/test/magic.py	2005-04-12 09:08:40.714909088 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 
 import sys
diff -ur webcleaner-2.29/test/parsefile.py webcleaner-2.29.new/test/parsefile.py
--- webcleaner-2.29/test/parsefile.py	2005-04-11 19:32:22.000000000 +0000
+++ webcleaner-2.29.new/test/parsefile.py	2005-04-12 09:08:40.714909088 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 
 def _main (args):
diff -ur webcleaner-2.29/test/pconfig.py webcleaner-2.29.new/test/pconfig.py
--- webcleaner-2.29/test/pconfig.py	2005-04-11 19:32:22.000000000 +0000
+++ webcleaner-2.29.new/test/pconfig.py	2005-04-12 09:08:40.715908936 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 """configuration loading profiling
 """
diff -ur webcleaner-2.29/test/pfilterfile.py webcleaner-2.29.new/test/pfilterfile.py
--- webcleaner-2.29/test/pfilterfile.py	2005-04-11 19:32:22.000000000 +0000
+++ webcleaner-2.29.new/test/pfilterfile.py	2005-04-12 09:08:40.715908936 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 """USAGE: test/run.sh test/pfilterfile.py <.html file>"""
 
diff -ur webcleaner-2.29/test/showprof.py webcleaner-2.29.new/test/showprof.py
--- webcleaner-2.29/test/showprof.py	2005-04-11 19:32:22.000000000 +0000
+++ webcleaner-2.29.new/test/showprof.py	2005-04-12 09:08:40.715908936 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 
 def _main (fname):
diff -ur webcleaner-2.29/test.py webcleaner-2.29.new/test.py
--- webcleaner-2.29/test.py	2005-04-11 19:32:22.000000000 +0000
+++ webcleaner-2.29.new/test.py	2005-04-12 09:08:00.122080136 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.3
+#!/usr/bin/python
 #
 # SchoolTool - common information systems platform for school administration
 # Copyright (c) 2003 Shuttleworth Foundation
